python/servo/testing_commands.py

```diff
@@ -203,6 +203,7 @@ def test_unit(self, build_type: BuildType, test_name=None, package=None, bench=F
             args += ["--", "--nocapture"]
 
         env = self.build_env()
+        env['CARGO_BUILD_RUSTC'] = 'crown'
```

Comment:
This will mean that people running `test-unit` locally will need to rebuild all of Servo so I think this should not be unconditional.

Reply:
Do we even need this? Crown unit-tests should be independent of this and others do not need crown anyway.

Reply:
> Do we even need this? Crown unit-tests should be independent of this and others do not need crown anyway.

You are right - the crown unit tests should not depend on the globally installed crown. They do fail if rustc is not overriden though, so I'll investigate what is going wrong in the unit tests.

Reply:
Interestingly I couldn't reproduce the unit-test failure locally anymore. Lets see what CI says.